pages/avatar/avatar.css

```diff
@@ -0,0 +1,45 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+*, ::before, ::after {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+}
+
+body {
+    font-family: Poppins, sans-serif;
+    background-color: rgb(0, 0, 0);
+    color: rgb(255, 255, 255);
+}
+
+.avatar-container{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
+}
+
+/* large */
+.avatar-large-image{
```

Comment:
Just a suggestion: You can name this class like this also: `.avatar-lg`. No need to write the image as this is the industry standard and followed by many. Similar for medium and small `avatar-md` & `avatar-sm` respectively.